css/properties.json

```diff
@@ -4772,6 +4743,52 @@
     "order": "uniqueOrder",
     "status": "standard"
   },
+  "offset-distance": {
+    "syntax": "&lt;length-percentage&gt;",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "lpc",
+    "percentages": "referToTotalPathLength",
+    "groups": [
+      "CSS Motion"
+    ],
+    "initial": "<code>0</code>",
+    "appliesto": "allElementsSVGContainerElements",
+    "computed": "asSpecified",
+    "order": "uniqueOrder",
+    "status": "experimental"
+  },
+  "offset-path": {
+    "syntax": "&lt;url&gt; | [ &lt;basic-shape&gt; | &lt;path()&gt; ] || &lt;geometry-box&gt; | none",
+    "media": "visual",
+    "inherited": false,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Motion"
+    ],
+    "initial": "<code>none</code>",
+    "appliesto": "allElementsSVGContainerElements",
+    "computed": "asSpecified",
+    "order": "uniqueOrder",
+    "stacking": true,
+    "status": "experimental"
+  },
+  "offset-rotation": {
+    "syntax": "[ auto | reverse ] &amp;&amp; &lt;angle&gt;",
```

Comment:
The [syntax](https://drafts.fxtf.org/motion-1/#propdef-offset-rotate) has pipes instead of ampersands now.

Sebastian